pkg/log/testdata/config.yml

```diff
@@ -0,0 +1,6 @@
+log:
+  level: warn
```

Comment:
What does this level mean? I can't find a use for that

Reply:
It's used in: 

`config.UnmarshalKey(handlerConfigKey, settings, cfg.UnmarshalWithDefaultsFromKey("log.level", "level"))`  
`gosoline/pkg/log/handler_iowriter.go:21`  

Enables one to configure the default level for all handlers in a single place.

Reply:
I see. Yeah, that makes sense, just looked a little bit confusing to have two different levels there without a direct explanation.